### PR TITLE
Show format string completion options when the initial quote is typed

### PR DESCRIPTION
**Pull request by jnm2**
This reminds me that the feature exists and that I don't need to search my way over to the docs to remember what I want.

fyi @CyrusNajmabadi

list_after_initial_quote

### CODE REVIEW

File: src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_DateAndTime.vb [hunk 1]
```diff
@@ -7,6 +7,47 @@ Imports System.Globalization
...
+{
+    void goo(DateTime d)
+    {
+        d.ToString(" $$);
+    }
+}
+]]></Document>, showCompletionInArgumentLists:=showCompletionInArgumentLists)
+
+                state.SendTypeChars("""")
+                Await state.AssertNoCompletionSession()
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_DateAndTime.vb:47]: awesome.  i was hoping you'd have this tes t:)

File: src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_DateAndTime.vb [hunk 2]
```diff
@@ -22,10 +63,10 @@ class c
 ]]></Document>, showCompletionInArgumentLists:=showCompletionInArgumentLists)
 
                 state.SendInvokeCompletionList()
-                Await state.AssertSelectedCompletionItem("d", inlineDescription:=FeaturesResources.short_date)
+                Await state.AssertSelectedCompletionItem("G", inlineDescription:=FeaturesResources.general_long_date_time)
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_DateAndTime.vb:66]: why did this cahnge?

File: src/Features/Core/Portable/EmbeddedLanguages/DateAndTime/DateAndTimeEmbeddedCompletionProvider.cs [hunk 3]
```diff
@@ -124,7 +131,7 @@ private static void ProvideStandardFormats(EmbeddedCompletionContext context)
             context.AddStandard("f", FeaturesResources.full_short_date_time, FeaturesResources.full_short_date_time_description);
             context.AddStandard("F", FeaturesResources.full_long_date_time, FeaturesResources.full_long_date_time_description);
             context.AddStandard("g", FeaturesResources.general_short_date_time, FeaturesResources.general_short_date_time_description);
-            context.AddStandard("G", FeaturesResources.general_long_date_time, FeaturesResources.general_long_date_time_description);
+            context.AddStandard("G", FeaturesResources.general_long_date_time, FeaturesResources.general_long_date_time_description, isDefault: true); // This is what DateTime.ToString() uses
```
Comment (CyrusNajmabadi) [inline src/Features/Core/Portable/EmbeddedLanguages/DateAndTime/DateAndTimeEmbeddedCompletionProvider.cs:134]: ok. let's try this out for now :)

**Comment by jnm2** [comment]
Why is this PR in the Completed column of https://github.com/dotnet/roslyn/projects/36 if it has not merged?

**Comment by jinujoseph** [comment]
@jnm2 that was a mistake, have corrected it. thanks for catching

**Comment by CyrusNajmabadi** [comment]
Awesome thanks!

**Comment by jnm2** [comment]
🎉 Thank you Jinu!

**Pull request merged**